server/src-lib/Hasura/GraphQL/Validate/Types.hs

```diff
@@ -171,11 +171,13 @@ type ParamMap = Map.HashMap G.Name InpValInfo
 -- | location of the type: a hasura type or a remote type
 data TypeLoc
   = TLHasuraType
-  | TLRemoteType !RemoteSchemaName !RemoteSchemaInfo
+  | TLRemoteType !RemoteSchemaName
```

Comment:
RemoteSchemaInfo seemed little too much data for this. This can be derived at the call-site perhaps.